src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```
@@ -45,8 +45,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AddPriFileDependsOn Condition="'$(MicrosoftPortableCurrentVersionPropsHasBeenImported)' == 'true'">DeterminePortableBuildCapabilities</AddPriFileDependsOn>
     <WarnOnPackingNonPackableProject Condition="'$(WarnOnPackingNonPackableProject)' == ''">false</WarnOnPackingNonPackableProject>
     <ImportNuGetBuildTasksPackTargetsFromSdk Condition="'$(ImportNuGetBuildTasksPackTargetsFromSdk)' == ''">false</ImportNuGetBuildTasksPackTargetsFromSdk>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>.dll; .exe; .winmd; .json; .pri; .xml; $(AllowedOutputExtensionsInPackageBuildOutputFolder)</AllowedOutputExtensionsInPackageBuildOutputFolder>
-    <AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder Condition="'$(SymbolPackageFormat)' != 'snupkg'">.pdb; .mdb; $(AllowedOutputExtensionsInPackageBuildOutputFolder); $(AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder)</AllowedOutputExtensionsInSymbolsPackageBuildOutputFolder>
+    <DefaultAllowedOutputExtensionsInPackageBuildOutputFolder Condition="'$(DefaultAllowedOutputExtensionsInPackageBuildOutputFolderCondition)' == ''">dll; .exe; .winmd; .json; .pri; .xml</DefaultAllowedOutputExtensionsInPackageBuildOutputFolder>
```

```suggestion
    <DefaultAllowedOutputExtensionsInPackageBuildOutputFolder Condition="'$(EnableDefaultOutputExtensionsInPackageBuildOutputFolder)' != 'false'">dll; .exe; .winmd; .json; .pri; .xml</DefaultAllowedOutputExtensionsInPackageBuildOutputFolder>
```

OR


```suggestion
    <EnableDefaultOutputExtensionsInPackageBuildOutputFolder Condition="'$(EnableDefaultOutputExtensionsInPackageBuildOutputFolder)' == ''">true</EnableDefaultOutputExtensionsInPackageBuildOutputFolder>
    <DefaultAllowedOutputExtensionsInPackageBuildOutputFolder Condition="'$(EnableDefaultOutputExtensionsInPackageBuildOutputFolder)' == 'true'">dll; .exe; .winmd; .json; .pri; .xml</DefaultAllowedOutputExtensionsInPackageBuildOutputFolder>
```

Similar to what .NET SDK targets use.

---

Introducing an extra ```EnableDefaultOutputExtensionsInPackageBuildOutputFolder``` wont give us any extra use and users will have to set 2 props to change the default extensions